services/ui-src/src/forms/mcpar/ebssei/ebssei.schema.ts

```diff
@@ -1,3 +1,9 @@
 import { object } from "yup";
+import { checkbox, text } from "utils/forms/schemas";
 
-export default object({});
+export default object({
+  "ebessei-1": checkbox(),
+  "ebessei-1-13-text": text(),
+  "ebessei-2": checkbox(),
+  "ebessei-2-7-text": text(),
```

Comment:
Take a look at `bedr.schema.ts` for how we handle these nested fields. As this is right now it will require those text fields even if you didn't select other.